spylib/token.py

```diff
@@ -263,6 +260,9 @@ async def execute_gql(
             else:
                 raise ValueError(f'GraphQL query is incorrect:\n{errorlist}')
 
+        if not suppress_errors and 'errors' in jsondata and len(jsondata['errors']) >= 1:
```

Comment:
Little suggestion for shorter:

```suggestion
        if not suppress_errors and len(jsondata.get('errors', [])) >= 1:
```